ga4gh/cli.py

```diff
@@ -63,6 +64,51 @@ def server_main():
         app.backend = backend
         app.run(host="0.0.0.0", port=args.port, debug=True)
 
+
+def server_benchmark():
+    parser = argparse.ArgumentParser(
+        description="GA4GH reference server benchmark")
+
+    args = parser.parse_args()
+    backend = Backend("ga4gh-example-data", WormtableVariantSet)
+    app.backend = backend
+    times = []
+
+    # Very heavy query: all calls on chromosome 2
+    # (11 pages, 90 seconds to fetch the entire thing)
+    initialRequest = ga4gh.protocol.GASearchVariantsRequest()
+    initialRequest.referenceName = '2'
+    initialRequest.variantSetIds = ['1000g_2013']
+    initialRequest.callSetIds = None
+    initialRequest.pageSize = 100
+    initialRequest.start = 0
+    initialRequest.end = 100000
+    initialRequestString = initialRequest.toJSONString()
+
+    # Repeat the query ~10x; if we compute average we should probably
+    # throw out at least the first one.
+    # TODO: more sophisticated statistics. Sometimes we want min(),
+    # sometimes mean = sum() / len(), sometimes other measures,
+    # perhaps exclude outliers...
+    for i in range(0, 10):
```

Comment:
Agreed; when I was chasing down a bug I made everything explicit, but it doesn't need to be shipped.